lib/src/core/theme/theme_color.dart

```diff
@@ -14,4 +14,11 @@ class ThemeColor {
   static const Color pink = Color.fromRGBO(247, 172, 170, 1);
   static const Color red = Color.fromRGBO(238, 48, 38, 1);
   static const Color shadow = Color.fromRGBO(0, 0, 0, 0.2);
+
+  // application colors
+  static const Color backgroundColor = Color(0xFF2D2B41);
```

Comment:
I think most of these colours are already defined here, "greenTextColor" is "Primary" for example. If not you can add them here and then add them to the ColorTheme of the AppTheme so these colors get provided by the ThemeProvider widget.

Reply:
@jorgelrj ok or maybe we should remove old color names used in noNetworkConnectionSnackBar.dart and use standard names used at the top of theme_color.dart :)

Reply:
Yeah I think if there are duplicates, just use the ones already defined here, so use the red for the background for example. But don't use it directly like ThemeColors.red, but use the ThemeProvider widget. This has a builder that passes the theme and you can use the colours from there.